### PR TITLE
Deployment fixes

### PR DESCRIPTION
**Pull request by KeltarHelviett**

### CODE REVIEW

File: deploy.bash [hunk 1]
```diff
@@ -278,7 +278,7 @@ else
 fi
 
 echo "8. Download JS... "
-if [[ $step =~ (^|,)6(,|$) || $step == "*" ]]; then
+if [[ $step =~ (^|,)8(,|$) || $step == "*" ]]; then
```
Comment (klenin) [inline deploy.bash]: This commit should be squashed with the previous one.

File: deploy.bash [hunk 2]
```diff
@@ -83,7 +83,7 @@ fi
 
 echo "2. Install cpan packages... "
 if [[ $step =~ (^|,)2(,|$) || $step == "*" ]]; then
-	cpan_packages=(DBI DBD::Firebird Algorithm::Diff Text::Aspell SQL::Abstract Archive::Zip
+	cpan_packages=(inc::Module::Install DBI DBD::Firebird Algorithm::Diff Text::Aspell SQL::Abstract Archive::Zip
```
Comment (klenin) [inline deploy.bash]: `cpanm` should already install all required dependencies. What is special about this one?

Comment (KeltarHelviett) [reply deploy.bash]:
```
cpanm Text::MultiMarkdown
...
Configuring Text-MultiMarkdown-1.000035 ... N/A
! Configure failed for Text-MultiMarkdown-1.000035. See /home/keltar/.cpanm/work/1527485029.3006/build.log for details.
```
build.log:
```
...
Searching Text::MultiMarkdown () on cpanmetadb ...
--> Working on Text::MultiMarkdown
Fetching http://www.cpan.org/authors/id/B/BO/BOBTFISH/Text-MultiMarkdown-1.000035.tar.gz
-> OK
Unpacking Text-MultiMarkdown-1.000035.tar.gz
Entering Text-MultiMarkdown-1.000035
Checking configure dependencies from META.yml
Checking if you have ExtUtils::MakeMaker 6.59 ... Yes (7.24)
Configuring Text-MultiMarkdown-1.000035
Running Makefile.PL
Can't locate inc/Module/Install.pm in @INC (you may need to install the inc::Module::Install module) (@INC contains: /home/keltar/perl5/lib/perl5/5.26.0/x86_64-linux-gnu-thread-multi /home/keltar/perl5/lib/perl5/5.26.0 /home/keltar/perl5/lib/perl5/x86_64-linux-gnu-thread-multi /home/keltar/perl5/lib/perl5 /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.26.0 /usr/local/share/perl/5.26.0 /usr/lib/x86_64-linux-gnu/perl5/5.26 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.26 /usr/share/perl/5.26 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at Makefile.PL line 2.
BEGIN failed--compilation aborted at Makefile.PL line 2.
...
```

File: deploy.bash (hunk 1 above)
Comment (KeltarHelviett) [reply deploy.bash]: Done.

File: deploy.bash (hunk 2 above)
Comment (KeltarHelviett) [reply deploy.bash]: Change inc::Module::Install to Module::Install.

File: deploy.bash [hunk 3]
```diff
@@ -83,7 +83,7 @@ fi
 
 echo "2. Install cpan packages... "
 if [[ $step =~ (^|,)2(,|$) || $step == "*" ]]; then
-	cpan_packages=(DBI DBD::Firebird Algorithm::Diff Text::Aspell SQL::Abstract Archive::Zip
+	cpan_packages=(Module::Install DBI DBD::Firebird Algorithm::Diff Text::Aspell SQL::Abstract Archive::Zip
```
Comment (klenin) [inline deploy.bash]: Add this link to commit message:
https://github.com/bobtfish/text-multimarkdown/issues/33

File: sql/interbase/init_relations.sql [hunk 4]
```diff
@@ -198,7 +198,8 @@ CREATE TABLE problems (
     run_method         SMALLINT DEFAULT 0,
     players_count      VARCHAR(200),
     statement_url      VARCHAR(200) DEFAULT '',
-    explanation_url    VARCHAR(200) DEFAULT ''
+    explanation_url    VARCHAR(200) DEFAULT '',
+    sub_path           VARCHAR(200) DEFAULT '' NOT NULL
```
Comment (klenin) [inline sql/interbase/init_relations.sql]: 1) Remove this commit from PR, it belongs to another
2) Remove NOT NULL, this column should be nullable
3) Rename to repo_path

**Pull request merged**